tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/RecordTests.cs

```diff
@@ -50,21 +52,24 @@ public void TestStartRecordInMemory()
                     HttpContext = httpContext
                 }
             };
-            controller.Start();
+            await controller.Start();
             var recordingId = httpContext.Response.Headers["x-recording-id"].ToString();
 
             var (fileName, session) = testRecordingHandler.RecordingSessions[recordingId];
 
             Assert.Empty(fileName);
         }
 
-        [Fact]
-        public void TestStopRecordingSimple()
+        [Theory]
+        [InlineData("recordings/TestStartRecordSimple.json")]
```

Comment:
Is this intentionally the same as the next test case?

Reply:
It's not though! 

```
Simple
vs
Simplé
```

Reply:
Ah I missed that 😆 
Thanks!